sdk/eventhubs/azure-messaging-eventhubs/vcpkg/Config.cmake.in

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: MIT
+
+@PACKAGE_INIT@
+
+include(CMakeFindDependencyMacro)
+find_dependency(azure-core-cpp "1.8.0")
```

Comment:
Would event hubs depend on azure-core-amqp, because amqp depends on core-cpp 1.9.0-beta.1.
Should this match that?

Reply:
amqp is not release yet, core 1.8.0 was the lates version this works on, any other version is not found by vcpkg

Reply:
Look at amqp `Config.cmake.in`, it lists core `1.9.0-beta.1` as a dependency. I am assuming the version of core that amqp needs and the version that eventhub needs, should match, because eventhubs will depend on amqp. Which is correct?

Reply:
Actually amqp needs 1.9.0-beta.1 because of the uuid changes. Eventhubs can't work with amqp **and** core 1.8.0. This should be updated to match.

Reply:
@gearama

Reply:
#4609